.github/workflows/release.yml

```
@@ -14,21 +14,18 @@ jobs:
         with:
           python-version: 3.9
-      - name: Install Poetry
-        run: |
-          pip install --upgrade pip
-          curl -sSL https://install.python-poetry.org | python -
+      - name: Install uv
+        uses: astral-sh/setup-uv@v4
       - name: Install dependencies
-        run: |
-          poetry install --no-dev
+        uses: uv sync --no-dev
```

actionlint (1.7.4)

specifying action "uv sync --no-dev" in invalid format because ref is missing. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)